packages/web-components/src/components/va-pagination/va-pagination.tsx

```diff
@@ -86,15 +87,73 @@ export class VaPagination {
    * Don't show last page when the page count exceeds
    * `maxPageListLength` (but do show the ellipsis).
    */
-
   @Prop() unbounded?: boolean = false;
 
+  /**
+   * Will be true when va-pagination is 480 pixels or narrower
+   * as measured by this.handleResizeEvent().
+   */
+  @State() isMobileViewport: Boolean = false;
+
+  /**
+   * Will be true when va-pagination is 640 pixels or narrower
+   * as measured by this.handleResizeEvent().
+   */
+  @State() isTabletViewport: Boolean = false;
+
   /**
    * If the page total is less than or equal to this limit, show all pages.
+   * The number has been reduced from 7 to 6 to show all more consistently
+   * on small screens and zoomed in browser windows.
+   */
+  SHOW_ALL_PAGES: number = 6;
+
+  /**
+   * Mobile viewport width chosen based on USWDS "mobile-lg" spacing unit.
+   * See https://designsystem.digital.gov/design-tokens/spacing-units/
```

Comment:
I think we would prefer this to reference VADS which also uses the same named/value `mobile-lg` token: https://design.va.gov/foundation/breakpoints

This can be updated in a few different places.